data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml

```diff
@@ -40,28 +42,31 @@
       </publicationStmt>
       
       <sourceDesc>
-        <listBibl xml:lang="lat">
+        <listBibl>
           <biblStruct>
-            <monogr>
+            <monogr>  
               <editor>
-                <persName>
-                  <name xml:lang="lat">Iosephus Zycha</name>
-                  <addName xml:lang="de">Joseph Zycha</addName>
-                </persName>
-              </editor> 
-              <author ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0040">Augustinus</author>  
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0040.stoa042">De Genesi ad Litteram Liber Imperfectus</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0040.stoa042a">De Genesi ad Litteram</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0040.stoa070">Locutiones in Heptateuchum</title>
+              <persName>
+                <name>Joseph Zycha</name>
+              </persName>
+            </editor> 
+              <author ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0040">Augustine</author>  
+              <title xml:lang="lat">Sancti Aureli Augustini Opera</title>            
               <imprint>
-                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                <pubPlace>Wien, Österreich</pubPlace>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
                 <date>1894</date>
               </imprint>
-              <biblScope unit="volume">28.1</biblScope>
+              <biblScope unit="volume" xml:lang="lat">Sectio III, Pars 1-2</biblScope>
```

Comment:
Latin? Also, I don't think that the Roman numerals are used elsewhere in this set. (I think some have Arabic numbers and some have Roman numerals.)

Reply:
I"m pretty sure I have been totally inconsistent across these files. What do you think is best @lcerrato that we have Section 3, Part 1-2,  or Sectio 3, Pars 1-2, and then just add the language tag? I do think you are right that I haven't included Roman numerals before. I like to be consistently inconsistent!

Reply:
@AlisonBabeu 
I think you just look at the rest of this collection and decide the best approach. Where it's Latin, I'm sure there are other Latin designations here. I'm sure that we use different designations throughout. 
I would lean towards keeping it as the book reads Latin + Roman numerals for ease of presentation. Just be sure the set is the same throughout.
(Note that the CTS files read **Sectio 3, Pars I-II** and this is the reverse **Sectio III, Pars 1-2**

Reply:
hi @lcerrato . Good plan. I will try to standardize these files and do a final pass through Augustine after I finish the last files.  I  think you are right and we should go with Latin and Roman numerals. What appears on the books seems to typically be Sect. VII Pars II, so they don't even spell out section. I do think I will stick with Arabic numbers for the volume numbers though, even though they also list those using Roman numerals.